.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,11 @@
+name: Docker Build
+on: [push]
```

Comment:
The last push of the pull request triggers it, for example for this pull request:

![image](https://user-images.githubusercontent.com/16060539/147933904-c2900a65-9ce9-477a-b35e-2fe85f542892.png)

Given it's useful to rebuild the container for each push, to check nothing broke, and given the last push triggers it when the pull request is open, this covers both.

Reply:
It does not, as people that create PRs do not have it running on their own repo. That is the purpose of PR Actions.

Please read more about it [here](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull-request-events-for-forked-repositories).

But essentially: 

> "Workflows don't run on forked repositories by default. You must enable GitHub Actions in the Actions tab of the forked repository.
When a first-time contributor submits a pull request to a public repository, a maintainer with write access may need to approve running workflows on the pull request. For more information, see "Approving workflow runs from public forks.""

Reply:
"Workflows don't run on forked repositories by default. You must enable GitHub Actions in the Actions tab of the forked repository" right, so it would not work. Forgot about forking pull requests. I will revert it then.